### ISSUE TITLE
False positives for no_implicit_braces with private function

### ISSUE THREAD

**Issue by provegard** [clutchski/coffeelint#606]
I have the following code:

```
class Foo

  privateFun = -> 42

  fun: -> privateFun()
```

I lint with the following config:

```
{
  "no_implicit_braces": {
    "level": "warn"
  }
}
```

And I get:

```
#5: Implicit braces are forbidden.
```

**Comment by swang** [comment]
Should be fixed now in latest 2.x versions.

**Comment by provegard** [comment]
Thanks, I'll test it!
